Examples/Swift/AppDelegate+CarPlay.swift

```diff
@@ -52,11 +52,13 @@ extension AppDelegate: CarPlayManagerDelegate {
             return nil
         }
         
+        let disableSimulateText = "Don’t Simulate"
+        let enableSimulateText =  "        Simulate"
```

Comment:
hmm, I wish we had something better here

Reply:
How about "Enable Simulation"/"Disable Simulation"

It's verbose (perhaps too long?), but it's clear, and this button doesn't need to share real estate (for now)

Reply:
@akitchen Sure. I will make those changes.